src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -7,6 +7,8 @@ import TurndownService = require('turndown');
 import { URI } from 'vs/base/common/uri';
 import * as path from 'vs/base/common/path';
 import * as turndownPluginGfm from 'sql/workbench/contrib/notebook/browser/turndownPluginGfm';
+import { invalidRelativePathRegex, replaceInvalidLinkPath } from 'sql/workbench/contrib/notebook/test/common/utils';
```

Comment:
We shouldn't be referencing utils from test folders

Reply:
moved to notebook/common